lib/ItemRow/stories/ItemRowStory.js

```diff
@@ -36,64 +38,112 @@ storiesOf('Components', module).add(
     }
 
     return (
-      <ItemRow
-        bordered={bordered}
-        stacked={stacked}
-        style={{
-          position: 'relative'
-        }}
-      >
-        {(draggedAbove || draggedBelow) && (
-          <DragLine alignment={alignment} offsetPx={2} />
-        )}
-        <ItemRow.Name>
-          {!stacked &&
-            createStatusIndicator(
-              object('Status', {
-                label: '',
-                color: 'green',
-                preText: '',
-                small: false,
-                softLabel: false
-              })
+      <>
+        <StoryItem title="ItemRow" description="The basic item row">
```

Comment:
Github is having a struggle here, this StoryItem hasn't changed, Its just a new one below has been added so everything had to be wrapped in `</>`s